src/handler/events/patternHandler.ts

```diff
@@ -0,0 +1,71 @@
+import { EMPTY, Observable, catchError, concatMap, finalize, fromEvent, map, of, pipe } from 'rxjs';
```

Comment:
Do you think merging this into messageHandler is a valid option? Or do you think separating patterns and message prefix commands is the better idea

Reply:
well I thought about both, proceeded with the separate one and got stuck, so I guess I could give other option a try?

Reply:
I'll see also, will get back to you